README.md

```diff
@@ -84,6 +84,7 @@ Additionally, we have designed every element of the Stack such that APIs as well
 |                                         Fireworks                                          |         Hosted         | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    |                    |
```

Comment:
### 📣 Reviewers - action required:

Don't review the current diff.

Please use review this diff:
https://github.com/meta-llama/llama-stack/pull/630/files/09ab5c8eabc5893942df036b762c478c3ebea230..HEAD 👈

Your comments will still appear on this PR

GitHub doesn't support PR trains to forks (the current diff contains previous PRs in the train)